pegasus/cache/i18n/vi-VN.yml

```diff
@@ -2121,15 +2120,3 @@
     Classroom” button on the “Manage students” tab:'
   google_sign_in_instruct: 'From there, they should choose the “Sign in with Google
     Account” option:'
-  og_title_soon: The Hour of Code is coming
```

Comment:
Seems weird these strings were removed.

Reply:
looks like they were just added yesterday and my sync didn't include that change

Reply:
However it also looks like the language is appropriately falling back to english, so I'm not worried about them missing

Reply:
I didn't know `I18n.t` falls back! Sweet. I'll just add to list of stuff to test in bug bash, just to be sure.